src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -33,13 +34,41 @@ public class NodeReadConverter : JsonConverter
     {
         private CustomNodeManager manager;
         private LibraryServices libraryServices;
+        private bool isTestMode;
+
         
         public ElementResolver ElementResolver { get; set; }
+        //map of all loaded assemblies including LoadFrom context assemblies
+        private Dictionary<string, List<Assembly>> loadedAssemblies;
 
-        public NodeReadConverter(CustomNodeManager manager, LibraryServices libraryServices)
+        public NodeReadConverter(CustomNodeManager manager, LibraryServices libraryServices, bool isTestMode = false)
         {
             this.manager = manager;
             this.libraryServices = libraryServices;
+            this.isTestMode = isTestMode;
+
+            if (this.isTestMode)
```

Comment:
can you specify why this is done only in test mode?

Reply:
I dont want to change the behavior of dynamo loading graphs - it will be slower and this is only a problem during testing.

Reply:
did you want me to add a comment to the code here?

Reply:
yes in the comment.